File: questPresentation/src/main/java/ru/nekit/android/qls/lockScreen/receiver/AlarmClockReceiver.java
package ru.nekit.android.qls.lockScreen.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import ru.nekit.android.qls.lockScreen.LockScreen;

public class AlarmClockReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        LockScreen.hide(context);
    }
}

File: questPresentation/src/main/java/ru/nekit/android/qls/lockScreen/receiver/AlarmClockReceiver.kt
package ru.nekit.android.qls.lockScreen.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

import ru.nekit.android.qls.lockScreen.LockScreen

class AlarmClockReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        LockScreen.hide(context)
    }
}
